index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Heptagram Bot</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Lexend+Deca&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/64fdbfc31c.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="header">
+        <img src="img/heptagram-logo.png" alt="heptagram-logo" class="logo">
+        <h1>Heptagram</h1>
+    </div>
+    <div class="description">
+        <h2>What is Heptagram ?</h2>
+        <h3>Heptagram is the <u>all-in-one</u>, <u>Open Source Discord Bot</u> with the goal to be the single bot needed bot for any server. 
+            <br>
+            No more, music bot, mod bot, rules bot. <u>Just Heptagram!</u>
+        </h3>
+        <img src="img/bot-menu.PNG" alt="bot-menu" class="bot-menu">
+    </div>
+    <div class="coming-soon">
+        <h4>This bot is <u>still in development</u>. We plan to come out with <u>V1 soon</u>.</h4>
+        <div class="social">
+            <div class="github-icon">
+                <a href="https://github.com/Heptagram-Bot">
+                    <i class="fab fa-github fa-2x"></i>
+                </a>
+            </div>
+            <div class="discord-icon">
+                <a href="https://discord.gg/HSupF99kpq">
+                    <i class="fab fa-discord fa-2x"></i>
+                </a>
+            </div>
+        </div>
+        <div class="attribution">
+            <p>Designed and coded by <a href="https://github.com/MidouWebDev">MidouWebDev</a> (one of the Heptagram Developers)</p>
```

Comment:
Hey @MidouWebDev, love giving you some credit, but can we also give some to the team? Maybe something like _**Designed and coded by the Heptagram Team**_?

Reply:
Also totally fine with keeping you in as well, just want to give some credit to the rest of the team. (Especially once we get the full site in)

Reply:
Okey, no problem ! I am planning to create a Heptagram Website Developers Page to give credits to everyone.